projects/rocprofiler-systems/source/lib/core/argparse.cpp

```diff
@@ -368,6 +368,19 @@ add_core_arguments(parser_t& _parser, parser_data& _data)
         _data.processed_environs.emplace("trace");
     }
 
+    if(_data.environ_filter("trace-legacy", _data))
+    {
+        _parser
+            .add_argument({ "-L", "--trace-legacy" },
+                          "Generate a detailed trace (perfetto output)")
```

Comment:
We should change the message here too.
```suggestion
                          "Generate a detailed trace (perfetto output) using the legacy tracing algorithm")
```